### PR TITLE
Update barren_plateaus.ipynb

### PR DESCRIPTION
**Pull request by abhinavsp0730**
changed this
```
try:
    %tensorflow_version 2.x
except Exception:
    pass
```
to
```
try:
     # %tensorflow_version only works in Colab.
    %tensorflow_version 2.x
except Exception:
    pass
```

### CODE REVIEW

**Comment by MichaelBroughton** [comment]
Based on the diff, I can't actually see that you've added the
```
# %tensorflow_version only works in Colab.
```
Could you also update all of the notebooks with your comment that the colab magic only works on colab ?

**Comment by abhinavsp0730** [comment]
Yaah, sure will do.

**Comment by abhinavsp0730** [comment]
Hey, @MichaelBroughton I've done the changes kindly review now. And I'm now going to update all the notebooks.

**Comment by abhinavsp0730** [comment]
Ya will do, just for the ease for reviewer I've done this.
Now I'm gonna add all the changes in one PR and will close the rest other.
Thank you.

**Comment by abhinavsp0730** [comment]
Hey, I've done all the changes that you've suggested.

File: docs/tutorials/gradients.ipynb [hunk 1]
```diff
@@ -126,6 +126,7 @@
       },
       "source": [
         "try:\n",
+         "    # %tensorflow_version only works in Colab.\n",
```
Comment (MichaelBroughton) [inline docs/tutorials/gradients.ipynb]: Extra space here

Comment (abhinavsp0730) [reply docs/tutorials/gradients.ipynb]: @MichaelBroughton I've corrected my formatting, kindly review it.

**Pull request merged**